java/gradle-plugin/README.md

```diff
@@ -1,6 +1,6 @@
 To build with a development version of this plugin, repeat as follows:
 * make changes
 * gradle build
-* mvn install:install-file -Dfile=${BOND_JAVA_ROOT}/gradle-plugin/build/libs/bond-gradle-1.0.jar -DgroupId=org.bondlib -DartifactId=gradle -Dversion=1.0 -Dpackaging=jar
+* mvn install:install-file -Dfile=build/libs/bond-gradle-1.0.jar -DgroupId=org.bondlib -DartifactId=gradle -Dversion=1.0 -Dpackaging=jar
```

Comment:
>build/libs/bond-gradle-1.0.jar [](start = 34, length = 30)

This assumes a particular working directory. Please add a comment specifying what that directory is.